src/routes/articlesRoutes/articlesRoutes.js

```diff
@@ -0,0 +1,14 @@
+module.exports = async function articleRoutes(fastify) {
+  fastify.get('/articles', async () => {
+    const client = await fastify.pg.connect()
+    const { rows } = await client.query(
+      `
```

Comment:
Bring the back-tick up one line so that it's joined with the opening parentheses.
```suggestion
    await client.query(`
        SQL QUERY HERE
    `)
```

This will help increase the amount of vertical space available